sdk/keyvault/azure-security-keyvault-keys/src/key_encryption_algorithm.cpp

```diff
@@ -1,17 +1,16 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
-
 #include "azure/keyvault/keys/key_client_models.hpp"
 #include "private/key_constants.hpp"
 
 namespace Azure { namespace Security { namespace KeyVault { namespace Keys {
 
-  const KeyEncryptionAlgorithm KeyEncryptionAlgorithm::CKM_RSA_AES_KEY_WRAP(
+  const KeyEncryptionAlgorithm KeyEncryptionAlgorithm::CkmRsaAesKeyWrap(
```

Comment:
Instead of renaming, let's also allocate the deprecated ones here?

This way, if customer still uses the now deprecated `KeyEncryptionAlgorithm::CKM_RSA_AES_KEY_WRAP`, they only get a warning from compiler, but not a linker error.

And if we want to be extra bulletproof, we could put (**only in this file**) `#undef CKM_RSA_AES_KEY_WRAP` and the same for the other two, before the first `#include`, so that even if CKM_RSA_AES_KEY_WRAP is defined when this very .cpp file is building, we forcefully undef it (and it should be safe to do so, because that undeffing will only affect building this .cpp file. But maybe undeffing is an overkill.

But I do recommend to still allocate `KeyEncryptionAlgorithm::CKM_RSA_AES_KEY_WRAP` so that if the customer does not have CKM_RSA_AES_KEY_WRAP, and uses a deprecated value, their code builds and links (even with the deprecation warning).

Reply:
when trying to initialize here i got build breaks due to the deprecated flag. 
i tried disabling the warning but that works only on msvc, 
i tried initializing it in class but that doesn't work for static const members . so the only choice i had was to remove the inits from here

Reply:
done